src/main/java/entities/SessionStorage.java

```diff
@@ -58,11 +59,53 @@ public MonthlyStorage getMonthlyData(int monthID) throws EntityException {
         throw new EntityException("That is not a valid currentMonth for this SessionStorage.");
     }
 
+    /**
+     * Returns the list of the MonthlyData objects stored in this Session
+     * @return list of all MonthlyData objects
```

Comment:
I think this part of the Javadoc should be written like "@return list ArrayList of all MonthlyData objects".

Make sure other Javadocs are also in the right format.